x-pack/test/functional/services/machine_learning/api.ts

```diff
@@ -247,5 +247,16 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
         }
       });
     },
+
+    async createCalendar(calendarId: string, body = { description: '', job_ids: [] }) {
+      await esSupertest
+        .put(`/_ml/calendars/${calendarId}`)
+        .send(body)
+        .expect(200);
+    },
```

Comment:
After this request we should wait for the calendar to be created as a response of `200` only acknowledges the request, but doesn't guarantee that the creation is done. See e.g. the `deleteIndices` method.

Reply:
Check https://github.com/elastic/kibana/pull/52269/commits/09334634271c5888e38c223250f7b8952bc4e500